rpcs3/Input/basic_keyboard_handler.cpp

```diff
@@ -194,7 +194,7 @@ void basic_keyboard_handler::LoadSettings()
 	//buttons.emplace_back(, CELL_KEYC_E_UNDEF);
 	buttons.emplace_back(Qt::Key_Escape, CELL_KEYC_ESCAPE);
 	buttons.emplace_back(Qt::Key_Kanji, CELL_KEYC_106_KANJI);
-	//buttons.emplace_back(Qt::Key_CapsLock, CELL_KEYC_CAPS_LOCK);
+	buttons.emplace_back(Qt::Key_CapsLock, CELL_KEYC_CAPS_LOCK);
```

Comment:
Keep in mind that this is not osk code.
This is cellKb code.
I'm pretty sure there was a reason for this.

Reply:
> Keep in mind that this is not osk code. This is cellKb code. I'm pretty sure there was a reason for this.

So, would the proper implementation create a separate callback similar to the shift callback?

Reply:
I'm not sure. Haven't looked at it yet

Reply:
I think there was a general issue in cellKb with how the shift modifier works.
But I guess it's not an issue until someone complains and I added a bunch of modifier fixes in the past anyway.